front/src/modules/timetableItem/components/Timetable/TimetableToolbar.tsx

```diff
@@ -249,7 +249,7 @@ const TimetableToolbar = ({
             'with-details': isInSelection,
           })}
         >
-          <div className="train-count" data-testid="timetable-train-count">
+          <div className="train-count" data-testid="timetable-items-count">
```

Comment:
nit : in english, `count` already implies that there will be multiple items
```suggestion
          <div className="train-count" data-testid="timetable-item-count">
```